src/Makefile

```diff
@@ -35,19 +38,24 @@ endif
 OBJECT_FILES= trexio.o trexio_text.o trexio_hdf5.o
 HEADER_FILES= trexio.h trexio_text.h trexio_hdf5.h trexio_s.h
 
-export CC CFLAGS LIBS
+export CC CFLAGS FC FFLAGS LIBS
 
 .PHONY: clean
 
 libtrexio.so: $(OBJECT_FILES) $(HEADER_FILES)
 	$(CC) -shared $(OBJECT_FILES) -o libtrexio.so
 
-test: libtrexio.so test.c
+fortran: libtrexio.so trexio_f.f90
+	$(FC) $(FFLAGS) -c trexio_f.f90 -o trexio_f.o
+
+test: libtrexio.so test.c test.f90 trexio_f.o
+	rm -f -r test_text_fort/
 	$(CC) $(CFLAGS) $(INCLUDE) -Wl,-rpath,$(PWD) -L. test.c -ltrexio $(LIBS) -o test
+	$(CC) $(CFLAGS) $(INCLUDE) -Wl,-rpath,$(PWD) -L. test.f90 trexio_f.o -ltrexio $(LIBS) -o test_f
 
 clean:
-	rm -f *.o libtrexio.so test test_*.h5 
-	rm -r trexio_test/
+	rm -f *.o libtrexio.so test test_*.h5 test_f
+	rm -r test_text_fort/
```

Comment:
```suggestion
	$(RM) *.o libtrexio.so test test_*.h5 test_f
	$(RM) -r test_text_fort/
```

Reply:
I don't see the definiton of $(RM) in Makefile. Should it be something like "rm -f" ?